lib/node_modules/@stdlib/stats/base/dists/hypergeometric/entropy/lib/main.js

```diff
@@ -0,0 +1,95 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var isNonNegativeInteger = require( '@stdlib/math/base/assert/is-nonnegative-integer' );
+var ln = require( '@stdlib/math/base/special/ln' );
+var pmf = require( '@stdlib/stats/base/dists/hypergeometric/pmf' );
+
+
+// MAIN //
+
+/**
+* Returns the Shannon entropy of a hypergeometric distribution.
+*
+* @param {NonNegativeInteger} N - population size
+* @param {NonNegativeInteger} K - subpopulation size
+* @param {NonNegativeInteger} n - number of draws
+* @returns {number} entropy
+*
+* @example
+* var h = entropy( 16, 11, 4 );
+* // returns ~1.216
+*
+* @example
+* var h = entropy( 2, 1, 1 );
+* // returns ~0.693
+*
+* @example
+* var h = entropy( 10, 5, 12 );
+* // returns NaN
+*/
+function entropy(N, K, n) {
```

Comment:
You should update your use of whitespace here and below in order to be consistent within this package and with the rest of stdlib.

```suggestion
function entropy( N, K, n ) {
```